### PR TITLE
Remove misplaced divider in Global Pipelines Page

### LINKED ISSUE #1606: [Bug]: Misplaced divider in the Global Pipelines page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Deploy type

OpenDataHub core version (eg. `v1.6.0`)

### Version

v2.13.0

### Current Behavior

There's a divider between the header and the Pipelines table on the Global Pipelines page. This is present on other pages too, but there are navigation tabs in place. We don't have any navigation tabs here.
Screenshot 2023-07-27 at 6 50 26 PM

### Expected Behavior

Since we don't have any navigation tabs on this page, we can remove the Divider just like what we have on the Projects page.

### Steps To Reproduce

1. Go to the Pipelines page by clicking on the same in the left Navigation bar.

### Workaround (if any)

_No response_

### What browsers are you seeing the problem on?

_No response_

### Anything else

_No response_

### PR DESCRIPTION
**Pull request by manaswinidas**
Fixes #1606 

## Description

This is how the Global Pipelines page look like now.
Screenshot 2023-08-04 at 2 54 15 PM

## How Has This Been Tested?

1. Go to the `Pipelines` page using the left Navbar
2. Check whether you can see the divider between the table and the header

## Test Impact

Visual changes.

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [X] Included any necessary screenshots or gifs if it was a UI change.
- [X] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by manaswinidas** [comment]
@kywalker-rh can you have a look at the screenshot above? Not sure if this is the ideal space between the header and the table so that the table dropdown and the header dropdown aren't confusing.

**Review by uidoyen — APPROVED** [review]
/lgtm

**Review by lucferbux — APPROVED** [review]
/lgtm

File: frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx [hunk 1]
```diff
@@ -20,7 +19,6 @@ const GlobalPipelines: React.FC = () => {
       headerAction={pipelinesAPi.pipelinesServer.installed && <PipelinesPageHeaderActions />}
       getRedirectPath={(namespace) => `/pipelines/${namespace}`}
     >
-      <Divider />
```
Comment (lucferbux) [inline frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx:23]: As @kywalker-rh petition, revamp the layout to remove the extra padding

**Review by lucferbux — CHANGES_REQUESTED** [review]

File: frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx (hunk 1 above)
Comment (manaswinidas) [reply frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx:23]: I double-checked with a Chrome extension(Smart page ruler) and checked that the padding was the same as the Projects page. Let me know if I missed something.
Screenshot 2023-08-07 at 4 35 31 PM
Screenshot 2023-08-07 at 4 36 01 PM
Screenshot 2023-08-07 at 4 30 41 PM
Screenshot 2023-08-07 at 4 31 01 PM

Comment (lucferbux) [reply frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx:23]: ha! you are completly right, I've just done the same with figma to verify. My bad, I just did a visual inspection turns out pipelines stick more cause it has the project selection and confused me.

@kywalker-rh I think we are fine here, I'm gonna add my lgmt

**Review by lucferbux — APPROVED** [review]
/lgtm

File: frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx (hunk 1 above)
Comment (manaswinidas) [reply frontend/src/pages/pipelines/global/pipelines/GlobalPipelines.tsx:23]: Same here, even I did a visual inspection and thought the same. The dropdown vs text confused me.

**Pull request merged**